### PR TITLE
feat(3047139090): hide the available balance if wallet is not connected

### PR DESCRIPTION
**Pull request by galuzan-starkware**
### Description of the Changes

The component `NetworkMenu />` will render the component `<TokenBalance />` only if `isLoggedIn = true` 

### Checklist

- [ ] Changes have been done against master branch, and PR does not conflict
- [ ] New unit / functional tests have been added (whenever applicable)
- [ ] Test are passing in local environment
- [ ] Docs have been updated
- [ ] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/260)

### CODE REVIEW

File: src/components/Features/Transfer/Transfer.js [hunk 1]
```diff
@@ -174,6 +175,7 @@ export const Transfer = () => {
     return (
       <NetworkMenu
         isDisabled={bridgeIsFull}
+        isLoggedIn={isLoggedIn}
```
Comment (dan-ziv) [inline src/components/Features/Transfer/Transfer.js:178]: Don't pass it as a prop but use the hook inside the `NetworkMenu` itself.

**Review by dan-ziv — CHANGES_REQUESTED** [review]

**Comment by dan-ziv** [comment]
@galuzan-starkware The description should describe the way of your solution and not what you already wrote in the title.

**Comment by galuzan-starkware** [comment]
> @galuzan-starkware The description should describe the way of your solution and not what you already wrote in the title.

done

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**